src/main/java/dal/fatih/todoly/TaskManager.java

```diff
@@ -1,155 +1,262 @@
+        String taskIdInput = scn.nextLine();
+        if (taskIdInput.length() < 3) {
+            System.out.println("Please enter min first three characters!!!");
+            return;
+        }
+        Task task = null;
 
-        System.out.print("Task Id :");
-        String taskId = scn.nextLine();
-        Task task = tasks.get(taskId);
-        if (task != null) {
-            System.out.println(task);
-            System.out.println("----------------------------------");
-        } else {
-            System.out.println("Task not found");
+        try {
+            connection = dbConnection.getConnection();
+
+            String sql = "SELECT taskId,title,description,duedate FROM tasks WHERE taskId like ?";
+            preparedStatement = connection.prepareStatement(sql);
+            preparedStatement.setObject(1, "%" + taskIdInput + "%");
+            resultSet = preparedStatement.executeQuery();
+        } catch (SQLException e) {
+            System.out.println(e);
+        }
+        try {
+            while (resultSet.next()) {
+                counter++;
+                UUID taskId = resultSet.getObject("taskId", java.util.UUID.class);
+                String title = resultSet.getString("title");
+                String description = resultSet.getString("description");
+                Date dueDate = resultSet.getObject("dueDate", java.sql.Date.class);
+                System.out.println("Task id : "+taskId);
+                System.out.println("Title : "+title);
+                System.out.println("Description : "+description);
+                System.out.println("Due date : "+dueDate);
+            }
+            if (counter<1) {
+                System.out.println("No task found");
+            }
+            preparedStatement.close();
+            resultSet.close();
+            connection.close();
+
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
         }
     }
 
     private void deleteTask() {
 
-        System.out.print("Task Id :");
-        String taskId = scn.nextLine();
-        Task task = tasks.get(taskId);
-        if (task != null) {
-            tasks.remove(taskId);
-            System.out.println(task.getTitle() + " titled task deleted");
-            writeTaskFile();
-        } else {
-            System.out.println("Task not found");
+        System.out.print("Task Id: ");
+        String taskIdInput = scn.nextLine();
+
+        try {
+            connection = dbConnection.getConnection();
+            String sql = "delete FROM tasks WHERE taskId=?";
+            preparedStatement = connection.prepareStatement(sql);
+            preparedStatement.setObject(1, taskIdInput);
+            int rows = preparedStatement.executeUpdate();
+
+            if (rows > 0) {
+                System.out.println("Task deleted");
+            } else {
+                System.out.println("No task found");
+            }

```

Comment:
Same here. Move this to `TaskRepository` in a method called `delete(String taskId)` that would return true or false according to if the item exists